ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+## Questions
+1. What is the DOM?
+DOM stands for Document Object Model. It's basically a way to edit elements in a website (using JavaScript to access and edit HTML and CSS).
```

Comment:
Would have liked to see a more elaborate answer on this one, however the answers is still good.